blueprints/instance-initializer-test/files/tests/unit/instance-initializers/__name__-test.js

```diff
@@ -0,0 +1,23 @@
+import Ember from 'ember';
+import { initialize } from '<%= dependencyDepth %>/instance-initializers/<%= dasherizedModuleName %>';
+import { module, test } from 'qunit';
+
+var appInstance, application;
+
+module('<%= friendlyTestName %>', {
+  beforeEach: function() {
+    Ember.run(function() {
+      application = Ember.Application.create();
+      appInstance = application.__deprecatedInstance__;
```

Comment:
This doesn't feels right, but it's the only way i've found of getting an application instance